validator/sawtooth_validator/state/config_view.py

```diff
@@ -0,0 +1,115 @@
+# copyright 2017 intel corporation
+#
+# licensed under the apache license, version 2.0 (the "license");
+# you may not use this file except in compliance with the license.
+# you may obtain a copy of the license at
+#
+#     http://www.apache.org/licenses/license-2.0
+#
+# unless required by applicable law or agreed to in writing, software
+# distributed under the license is distributed on an "as is" basis,
+# without warranties or conditions of any kind, either express or implied.
+# see the license for the specific language governing permissions and
+# limitations under the license.
+# ------------------------------------------------------------------------------
+
+import hashlib
+
+from sawtooth_validator.protobuf.setting_pb2 import Setting
+
+
+class ConfigView(object):
+    """
+    A ConfigView is a snapshot view of on-chain configuration settings.
```

Comment:
Perhaps: "A ConfigView provides access to on-chain configuration stored at a particular merkle root.".